### PR TITLE
Remove groupby_simple_monitor parameter

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Remove the `groupby_simple_monitor` from the curl examples in the docs repo. This is deprecated
- [DOCS-8881](https://datadoghq.atlassian.net/browse/DOCS-8881)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-8881]: https://datadoghq.atlassian.net/browse/DOCS-8881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/monitors/configuration/_index.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ For more information on the configuration options for the notification message,
 
 ### Set alert aggregation
 
-Alerts are grouped automatically based on your selection of the `group by` step when defining your query. If the query has no grouping, it defaults to `Simple Alert`. If the query is grouped by any dimension, grouping changes to `Multi Alert`.
+Alerts are grouped automatically based on your selection of the aggregation selected for your query (for exanple, `avg by service`). If the query has no grouping, it defaults to `Simple Alert`. If the query is grouped by any dimension, grouping changes to `Multi Alert`.
```
Comment (drichards-87) [inline content/en/monitors/configuration/_index.md:254]:
```suggestion
Alerts are grouped automatically based on your selection of the aggregation selected for your query (for example, `avg by service`). If the query has no grouping, it defaults to `Simple Alert`. If the query is grouped by any dimension, grouping changes to `Multi Alert`.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Looks good. I left one small bit of feedback re: a spelling typo. I'll go ahead and commit that for you, and merge the PR after the build job finishes running.

**Review by phillip-dd — APPROVED** [review]
lgtm thanks

**Review by drichards-87 — APPROVED** [review]
Looks good!

**Pull request merged**